server/plugin/client.go

```diff
@@ -135,6 +135,29 @@ func (c *client) callFormURLEncoded(url, path, method string, out interface{}, f
 	return c.Call(url, method, path, contentType, "", nil, out, formValues)
 }
 
+var publisherID = map[string]string{
```

Comment:
you should add some comments explaining why we needed this in brief so that in future other devs can get this